### DISCUSSION TITLE
引用其它项目的组件时怎样区分不同环境？

### DISCUSSION THREAD

**Discussion by yesay** [empjs/emp#311] [Q&A] [answered]
怎样实现这里的地址根据不同的环境来区分不同的url地址？

![2022-09-02_170010](https://user-images.githubusercontent.com/3607509/188104433-ae0229c0-0602-4c3c-859b-2e4dfcb229fd.png)

**Comment by ckken** [comment, marked as answer]
https://juejin.cn/post/6968803470895742983
